### ISSUE TITLE
cmake/usd.cmake breaking Houdini Solaris environment.

### ISSUE THREAD

**Issue by suekieza** [Autodesk/maya-usd#85]
Hey there, 
How are "environment variables set in linux centos 7 using cmake/usd.cmake", I would like to turn them off. It is breaking Houdini Solaris USD on my Linux at the moment having these paths in my computers environments.

Thanks Sue

**Comment by BigRoy** [comment]
How are you seeing Houdini Solaris USD breaking?

The CMAKE environment variables should only be set for the running build process and should not influence any global system environment variables. As such, I'm not sure how the `usd.cmake` is breaking your system.

At first sight the issue seems quite unrelated to `maya-usd` so some more information would be welcome to understand how these issues would be linked.

**Comment by HamedSabri-adsk** [comment]
@suekieza do you still have PATH and PYTHONPATH in your .bashrc that you mentioned here:
https://github.com/Autodesk/maya-usd/issues/62#event-2743161704

If yes, you need to remove them.  cmake/usd.cmake shouldn't really have any global affect on your system.

Are you running running Houdini from the same shell you built maya-usd?  Try running Houdini from a different shell.

**Comment by suekieza** [comment]
Hey there,
@HamedSabri-adsk  I did indeed remove the PATH and PYTHONPATH from the .bashrc. I had to rename the USD file(on disk), as it was still linking to that file in the global environment variables. I'm not in the office today, will share images of what I mean with you guys tomorrow. It is probably just me doing something stupid.

@BigRoy When I open Solaris stage graph view in houdini it actually gives me an error saying that the USD PATH is incorrect, will share the msg with you tomorrow.
